src/components/sidebar/sidebar.component.js

```diff
@@ -6,9 +6,15 @@ import SidebarStyle from "./sidebar.style";
 import focusTrap from "../../utils/helpers/focus-trap";
 import IconButton from "../icon-button";
 import Icon from "../icon";
+import Browser from "../../utils/helpers/browser";
 
 class Sidebar extends Modal {
   /** Returns classes for the component. */
+  constructor(args) {
+    super(args);
+    this.document = Browser.getDocument();
```

Comment:
do we need that to access the `document`?

Reply:
yeah we do, If we just want to get to the document without this one there is an issue